instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/constants.py

```diff
@@ -0,0 +1,121 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEFAULT_LOGGING_FORMAT = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] [trace_id=%(otelTraceID)s span_id=%(otelSpanID)s resource.service.name=%(otelServiceName)s] - %(message)s"
+
+
+_MODULE_DOC = """
+The OpenTelemetry ``logging`` integration automatically injects tracing context into log statements.
+
+The integration patches the standard library logging module to automatically inject tracing context into log record objects
+and optionally calls ``logging.basicConfig()`` to set a logging format with placeholders for span ID, trace ID and service name.
+
+The following keys are injected into the patched log record objects:
+
+- ``otelSpanID``
+- ``otelTraceID``
+- ``otelServiceName``
+
+The integration uses the following logging format by default:
+
+.. code-block::
+
+    {default_logging_format}
+
+Enable trace context injection
+------------------------------
+
+The integration is opt-in and must be enabled explicitly by setting the environment variable ``OTEL_PYTHON_LOG_CORRELATION`` to ``true``.
+
+The integration always patches the logging module to inject tracing context into the log record objects. Setting ``OTEL_PYTHON_LOG_CORRELATION`` to ``true``
+calls ``logging.basicConfig()`` to set a logging format that actually makes use of the injected variables.
+
+
+Environment variables
+---------------------
+
+OTEL_PYTHON_LOG_CORRELATION
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This env var must be set to ``true`` in order to enable trace context injection into logs by calling ``logging.basicConfig()`` and
```

Comment:
nit they can also set it in `LoggingInstrumentor` constructor which I think you mention elsewhere, but this is misleading